src/gui/FileBrowser.cpp

```diff
@@ -380,6 +381,11 @@ void FileBrowserTreeWidget::contextMenuEvent(QContextMenuEvent * e )
 								"B+B Editor" ),
 						this,
 					SLOT( openInNewInstrumentTrackBBE() ) );
+		contextMenu.addSeparator();
+		contextMenu.addAction( QIcon(embed::getIconPixmap( "folder" )),
```

Comment:
Clean these four lines up following [Coding Conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions)

Reply:
Done with commit b85aef2f33794eb6bd7fa1153f7c411ef14c5b7c. I have cleaned up the whole method.

Reply:
Usually we try not to touch other lines unless there is a need, as eventually we'll have an autoformatter whenever that PR gets merged, but I think this is okay for now.